components/python/import-profiler/import-profiler.license

```diff
@@ -0,0 +1,14 @@
+Copyright 2017 David Cournapeau
```

Comment:
License file should have been named import_profiler.license, as it's searched as $(COMPONENT_NAME).license